lib/widgets/task_card.dart

```diff
@@ -22,7 +42,25 @@ class TaskCard extends StatelessWidget {
     return Dismissible(
       key: Key(task.id),
       direction: DismissDirection.endToStart,
-      background: Container(
+      background: _background(cardMargin),
+      onDismissed: onDismissSwap,
+      child: Card(
+        margin: cardMargin,
+        child: InkWell(
+          onTap: () => _onCardTap(context),
+          child: Row(
+            children: [
+              _completedCheckbox(),
```

Comment:
```suggestion
              _buildCompletedCheckbox(),
```
В именах методов должен быть глагол